app/src/androidTest/java/org/mozilla/reference/browser/ui/SettingsViewTest.kt

```diff
@@ -42,7 +42,8 @@ class SettingsViewTest {
             verifyMakeDefaultBrowserButton()
             verifyDeveloperToolsHeading()
             verifyRemoteDebuggingText()
-            verifyRemoteDebuggingToggle()
+            // There are 2 toggles with the same id, the following will always fail the test
+            // verifyRemoteDebuggingToggle()
```

Comment:
Can you please create an issue to track this as well?  Thanks,

Reply:
Are you sure we need an issue? This is how preference inflation works: every single `SwitchPreference` will end up having a `Switch` instance with the same ID, so `verifyRemoteDebuggingText` should be enough. We should add a `verifyOpenLinksInAppsText`, currently this is not covered by the test.

Reply:
I see.  Understood.  Do you want to open an issue to add `verifyOpenLinksInAppsTest`? to fix this commented out test case?  Thanks